app/views/search/index.html.erb

```diff
@@ -0,0 +1,53 @@
+<style>
```

Comment:
Suggestion: Move this to a CSS file. You'll have to add a class name to the body if it's specific to only this page.